libs/designsystem/src/lib/components/icon/icon.component.spec.ts

```diff
@@ -47,6 +47,22 @@ describe('IconComponent', () => {
       expect(component.name).toBe(undefined);
       expect(component.icon.name).toBe(component.defaultIcon.name);
     });
+
```

Comment:
Consider adding test that describes what happens if icon with same name is added BOTH through IconSettings AND Registry...

Reply:
Impl. as we talked about. IconSettings and Registry icons is merged together with warn on overwrite